src/options.rs

```diff
@@ -136,17 +145,29 @@ impl Options {
   }
 
   pub(crate) fn bitcoin_rpc_client_mainnet_forbidden(&self, command: &str) -> Result<Client> {
-    let client = self.bitcoin_rpc_client()?;
-
     if self.chain() == Chain::Mainnet {
       bail!("`{command}` is unstable and not yet supported on mainnet.");
     }
+
+    let client = self.bitcoin_rpc_client_for_wallet_command(command)?;
+
     Ok(client)
   }
 
   pub(crate) fn bitcoin_rpc_client_for_wallet_command(&self, command: &str) -> Result<Client> {
     let client = self.bitcoin_rpc_client()?;
 
+    const MIN_VERSION: usize = 240000;
+
+    let bitcoin_version = client.version()?;
+    if bitcoin_version < MIN_VERSION {
+      bail!(
+        "Bitcoin Core {} or newer required, current version is {}",
+        Self::format_bitcoin_core_version(MIN_VERSION),
+        Self::format_bitcoin_core_version(bitcoin_version),
+      );
+    }
+
     if self.chain() == Chain::Mainnet {
       let wallet_info = client.get_wallet_info()?;
```

Comment:
Only perform this check if the command is not ord wallet create, so, we don't break mainnet.